Bottom: 4b114224ace8e4bd636941cfd45dd49af42a86d5
Top:    629f19581b5dd714698d23ba4359b7b92afccf70
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-27 20:35:04 +0100

Make --enable-gl default when building the GTK HID


---

diff --git a/configure.ac b/configure.ac
index 2637495..94c6779 100644
--- a/configure.ac
+++ b/configure.ac
@@ -393,8 +393,12 @@ fi
 AC_MSG_CHECKING([for whether to use GL drawing])
 AC_ARG_ENABLE([gl],
 [  --enable-gl           Enable GL drawing (with GTK HID)],
-[],[enable_gl=no])
-
+[],[
+	case " $with_gui " in
+		*\ gtk\ *) enable_gl=yes;;
+		* ) enable_gl=no;;
+	esac
+])
 AC_MSG_RESULT([$enable_gl])
 AM_CONDITIONAL(USE_GL, test x$enable_gl = xyes)
